### PR TITLE
fix randint validation in mock-observations plugin

### PR DESCRIPTION
**Pull request by manushak**
### Types of changes

- [ ] Enhancement (project structure, spelling, grammar, formatting)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- add validation to check if the min is below the max of the randint config
-

### CODE REVIEW

**Review by jmcook1186 — APPROVED** [review]
lgtm

File: src/__tests__/unit/lib/mock-observations/index.test.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe('lib/mock-observations: ', () => {
             'common-key': 'common-val',
           },
           randint: {
-            'cpu/utilization': {min: 10, max: 10},
+            'cpu/utilization': {min: 10, max: 11},
```
Comment (narekhovhannisyan) [inline src/__tests__/unit/lib/mock-observations/index.test.ts:45]: @manushak you can add one more test case where min is more than max

File: src/lib/mock-observations/helpers/rand-int-generator.ts [hunk 2]
```diff
@@ -46,6 +46,14 @@ export const RandIntGenerator = (
...
     }
+
+    if (config.min >= config.max) {
+      throw new InputValidationError(
+        errorBuilder({
+          message: `Min value should not be greater than max value of ${validatedName}`,
+        })
+      );
+    }
     return {min: config.min, max: config.max};
```
Comment (narekhovhannisyan) [inline src/lib/mock-observations/helpers/rand-int-generator.ts:57]: I know that it's Paz's code, however, min and max can be used with restructuring and Object.prototype.hasOwnProperty can be replaced by 'min' in config, 'max' in config

File: src/lib/mock-observations/helpers/rand-int-generator.ts [hunk 3]
```diff
@@ -46,6 +46,14 @@ export const RandIntGenerator = (
         })
       );
     }
+
+    if (config.min >= config.max) {
+      throw new InputValidationError(
+        errorBuilder({
+          message: `Min value should not be greater than max value of ${validatedName}`,
```
Comment (MariamKhalatova) [inline src/lib/mock-observations/helpers/rand-int-generator.ts:53]: @manushak As we don't accept the equality between min and max, then let's rephrase the err. message to `Min value should not be greater than or equal to max value of ${validatedName}`

**Review by MariamKhalatova — APPROVED** [review]
Tested

**Pull request merged**